src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/cancelspinlock_bug1.tt.sarif

```diff
@@ -7,7 +7,7 @@
         "name": "StaticDriverVerifier"
       },
       "files": {
-        "..\\..\\..\\rules\\WDM\\CancelSpinLock.slic": {
+        "rules/WDM/CancelSpinLock.slic": {
```

Comment:
Darn it, sorry, I didn't see this comment before I merged. I'll add it to the next PR.